### PR TITLE
Math support recipe

### PR DESCRIPTION
**Pull request by TaiChi-IO**
add a recipe for supporting math formulas on the published github pages.

### CODE REVIEW

File: docs/math-support.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        }
+    });
+</script>
+```
+
+ If you want the index page of your Foam site to render maths, you'll need to add that to `_layouts/home.html` as well, or change the layout of the index page to be "page" instead of "home" by putting this Front Matter on the top of your `readme.md/index.md`:
+
+```
+---
+layout: home
```
Comment (riccardoferretti) [inline docs/math-support.md:21]: you mean `page`?

File: docs/math-support.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Math Support
+
+The published pages don't support math formulas by default. To enable this feature, you can add the following code snippet to the end of `_layouts/page.html`:
```
Comment (riccardoferretti) [inline docs/math-support.md:3]: I like the trick, would it be helpful to provide an example on how to write a math formula in markdown? Possibly with a link to the reference syntax?

Comment (jevakallio) [reply docs/math-support.md:3]: @TaiChi-IO agreed with @riccardoferretti, could you provide an example of the syntax you would use?

File: docs/math-support.md (hunk 1 above)
Comment (TaiChi-IO) [reply docs/math-support.md]: yes, please see the commit below https://github.com/foambubble/foam/pull/109/commits/e0cf333c591abacbb11d856f4ccb012d74de6c9a

File: docs/math-support.md (hunk 2 above)
Comment (TaiChi-IO) [reply docs/math-support.md:3]: yes, please see commit https://github.com/foambubble/foam/pull/109/commits/525b8cf4039df841b068b48bfee3910753e86f07

**Comment by jevakallio** [comment]
@all-contributors add @TaiChi-IO for doc

**Pull request merged**